doc/src/selections.rst

```diff
@@ -83,9 +95,14 @@ equivalent to ``atoms: name == H``.
 Then, if no context is given, the ``atoms`` context is used. This make ``atoms:
 name == H`` equivalent to ``name == H``.
 
-And finally, the ``==`` comparison operator is the default one if no operator is
+Then, the ``==`` comparison operator is the default one if no operator is
 precised. This means that we can write ``name H`` instead of ``name == H``.
 
+Finally, multiple values are interpreted as multiple choices. A selection like
```

Comment:
Remove "finally" as you did previously. You can place it at the beginning of the next paragraph instead of 'At the end' because this paragraph is a culmination of all the selection rules you've discussed previously.